src/main.ts

```diff
@@ -24,36 +18,15 @@ AWSXRay.captureHTTPsGlobal(https, true);
 //Capture all promises that we make
 AWSXRay.capturePromise();
 
-Sentry.init({
-  ...config.sentry,
-  debug: config.sentry.environment == 'development',
-});
-
-// Start BatchDelete queue polling
-new BatchDeleteHandler(new EventEmitter());
-
-const server = getServer();
-
-const app = express();
-
-app.use(express.json());
-app.use('/queueDelete', queueDeleteRouter);
-
-//If there is no host header (really there always should be..) then use parser-wrapper as the name
-app.use(xrayExpress.openSegment(serviceName));
-
 //Set XRay to use the host header to open its segment name.
 AWSXRay.middleware.enableDynamicNaming('*');
 
-//Apply the GraphQL middleware into the express app
-server.start().then(() => {
-  server.applyMiddleware({ app, path: '/' });
-});
-
-//Make sure the express app has the xray close segment handler
-app.use(xrayExpress.closeSegment());
+// init BatchDeleteHandler outside server, SQS queue is not
+// present in localstack for integration testing
+// Start BatchDelete queue polling
+new BatchDeleteHandler(new EventEmitter());
```

Comment:
As far as I can tell, SQS is not set up in localstack, and there's nothing to poll while running integration tests.  Is this true?